editor/debugger/editor_profiler.cpp

```diff
@@ -502,9 +509,34 @@ void EditorProfiler::_graph_tex_input(const Ref<InputEvent> &p_ev) {
 				frame_delay->start();
 			}
 		}
+	}
+
+	if (mb.is_valid() && (mm->get_button_mask().has_flag(MouseButtonMask::MIDDLE) || mm->get_button_mask().has_flag(MouseButtonMask::RIGHT)) && graph_zoom > 0) {
```

Comment:
```suggestion
	if (graph_zoom > 0 && mb.is_valid() && (mm->get_button_mask().has_flag(MouseButtonMask::MIDDLE) || mm->get_button_mask().has_flag(MouseButtonMask::RIGHT))) {
```
Cheaper condition first